lib/keyboard_utils.coffee

```diff
@@ -60,6 +62,7 @@ KeyboardUtils =
       if (event.keyCode >= @keyCodes.f1 && event.keyCode <= @keyCodes.f12)
         return "f" + (1 + event.keyCode - keyCodes.f1)
       return ""
+    return "backspace" if event.keyIdentifier == "U+0008"
```

Comment:
I've no enough knowledge about the line `if (event.keyIdentifier.slice(0, 2) != "U+")` - it has been there since commit 50a4fd9, with a comment `Not a letter` by Phil Crosby.

But it seems safe (or little harmful) to remove / delay this statement, for `keyIdentifier` has been deprecated and is to be removed on Chrome 54.

Do I need to make a new commit to delay it?